changelog/unreleased/2900

```diff
@@ -0,0 +1,7 @@
+Enhancement: Join users and groups into a single list in collaborators sidebar
```

Comment:
I think this is rather a `Change`, not an enhancement ?
see changelog/README for the possible values

Reply:
Agreed, I have edited the commit 👍